worlds/dark_souls_3/__init__.py

```diff
@@ -287,7 +290,7 @@ def generate_output(self, output_directory: str):
             "itemsAddress": items_address
         }
 
-        # generate the file
-        filename = f"AP-{self.world.seed_name}-P{self.player}-{self.world.player_name[self.player]}.json"
-        with open(os.path.join(output_directory, filename), 'w') as outfile:
-            json.dump(data, outfile)
+        return slot_data
+
+    def generate_output(self, output_directory: str):
+        pass
```

Comment:
you don't have to override it if it's empty
```suggestion
```

Reply:
In fact you really shouldn't as the empty override spawns a thread worker